test/e2e/dataplane/tcp_pod_to_service.go

```diff
@@ -30,51 +30,51 @@ var _ = Describe("[dataplane] Basic Pod to Service tests across clusters without
 	})
 })
 
-func testPod2ServiceTCP(f *framework.Framework, leftScheduling framework.TestPodScheduling, rightScheduling framework.TestPodScheduling) {
+func testPod2ServiceTCP(f *framework.Framework, leftScheduling framework.NetworkPodScheduling, rightScheduling framework.NetworkPodScheduling) {
```

Comment:
I like "TestPod..." more as it is generic. NetworkPod... becomes more specific which at this point is not needed IMO

Reply:
Yeah,  there was a discussion on the other patch around this.

TestPod is shorter, I agree with you, and we can assume that pods are going to be "network" always for us... but I'm ok with the current patch, we need to make progress. And thanks tom for proactively proposing the amendments.

Reply:
I'm OK either way - just trying to address Mike's concerns. While `TestPod` is generic, these pods really aren't generic, ie they run specific commands to test networking so `NetworkPod` seems more meaningful in this case.